array-manipulation-1/src/initial.js

```diff
@@ -13,10 +13,9 @@ function initial(array) {
     return [];
   }
   var newArray = [];
-  var lastIndex = array.length - 1;
 
   for (var i = 0; i < array.length; i++) {
-    if (i !== lastIndex) {
+    if (i < array.length - 1) {
```

Comment:
I meant change the condition in the loop configuration. Rather than continuing while `i < array.length`, if you continue while `i < array.length - 1`, you won't need the conditional.

Reply:
You can remove this `if` statement entirely.